include/qdmi/client.h

```diff
@@ -981,6 +1055,329 @@ void QDMI_job_free(QDMI_Job job);
+ * @details If not noted otherwise, parameters are mandatory and drivers must
+ * require them to be set.
+ */
+
+enum QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_T {
+
+  /// The start time of the telemetry query interval.
+  QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_STARTTIME = 0,
+
+  /// The end time of the telemetry query interval.
+  QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_ENDTIME = 1,
+
+  /// The telemetry for the telemetry query.
+  QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_TELEMETRY = 2,
+
+  /// The maximum value of the enum.
+  QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_MAX = 3,
+
+  /**
+   * @brief This enum value is reserved for a custom parameter.
+   * @details The driver defines the meaning and the type of this parameter.
+   * @attention The value of this enum member must not be changed to maintain
+   * binary compatibility.
+   */
+  QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_CUSTOM1 = 999999995,
+
+  /// @see QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_CUSTOM1
+  QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_CUSTOM2 = 999999996,
+
+  /// @see QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_CUSTOM1
+  QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_CUSTOM3 = 999999997,
+
+  /// @see QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_CUSTOM1
+  QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_CUSTOM4 = 999999998,
+
+  /// @see QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_CUSTOM1
+  QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_CUSTOM5 = 999999999
+};
+
+/// Telemetry sensor query parameter type.
+typedef enum QDMI_TELEMETRYSENSOR_QUERY_PARAMETER_T
+    QDMI_TelemetrySensor_Query_Parameter;
+/**
+ * @brief Set a parameter for an telemetry sensor.
+ * @param[in] query A handle to a job for which to set @p param. Must not be @c
+ * NULL.
+ * @param[in] param The parameter whose value will be set. Must be one of the
+ * values specified for @ref QDMI_TelemetrySensor_Query_Parameter.
+ * @param[in] size The size of the data pointed to by @p value in bytes. Must
+ * not be zero, except when @p value is @c NULL, in which case it is ignored.
+ * @param[in] value A pointer to the memory location that contains the value of
+ * the parameter to be set. The data pointed to by @p value is copied and can be
+ * safely reused after this function returns. If this is @c NULL, it is ignored.
+ * @return @ref QDMI_SUCCESS if the driver supports the specified @ref
+ * QDMI_TelemetrySensor_Query_Parameter @p param and, when @p value is not @c
+ * NULL, the parameter was successfully set.
+ * @return @ref QDMI_ERROR_NOTSUPPORTED if the driver does not support the
+ * parameter or the value of the parameter.
+ * @return @ref QDMI_ERROR_INVALIDARGUMENT if
+ *  - @p query is @c NULL,
+ *  - @p param is invalid, or
+ *  - @p value is not @c NULL and @p size is zero or not the expected size for
+ *  the parameter (if specified by the @ref
+ * QDMI_TelemetrySensor_Query_Parameter documentation).
+ * @return @ref QDMI_ERROR_BADSTATE if the parameter cannot be set in the
+ * current state of the job, for example, because the query is already
```

Comment:
```suggestion
 * current state of the query, for example, because the query is already
```
May you also check with the search function for more copy & paste mistakes, please?